config/config.go

```diff
@@ -301,9 +314,16 @@ func (config *MixConfig) Print() error {
 	return nil
 }
 
-func (config *MixConfig) initConfigPath(path string) error {
-	if path != "" {
-		config.filename = path
+func (config *MixConfig) initConfigPath(pathPrefix string) error {
+	if pathPrefix != "" {
+		// This must be a file, so use the path name directly
```

Comment:
instead of overloading `pathPrefix` to *also* mean `fullPath` just remove all this logic and require the argument to actually be the path prefix. If you need both capabilities for some reason add a `fullPath` argument or something.

Reply:
It should support passing in just "builder.conf" when you're in the current working directory though, passing a full path in that case would be annoying.

Reply:
so just make that the default if it is an empty string. Otherwise the argument does two different things based entirely on the suffix of the prefix...

Reply:
Renamed it to not path prefix :)